host/xtest/sdp_basic.c

```diff
@@ -338,10 +335,10 @@ static int dump_sdp_data(struct tee_ctx *ctx,
 		  void *out, size_t offset, size_t len, void *shm_ref, int ind)
 {
 	TEEC_SharedMemory *shm = (TEEC_SharedMemory *)shm_ref;
-	TEEC_Result teerc;
-	TEEC_Operation op;
-	uint32_t err_origin;
-	unsigned cmd;
+	TEEC_Result teerc = TEEC_ERROR_GENERIC;
+	TEEC_Operation op = TEEC_OPERATION_INITIALIZER;
+	uint32_t err_origin = 0;
+	unsigned cmd = 0;
```

Comment:
Please make this `unsigned int` while you're at it.

Reply:
ok, here and line 301 above.